src/dronevis/detection_torch/yolov5_torch.py

```diff
@@ -15,18 +17,27 @@ class YOLOv5(CVModel):
     For more details see `YOLOv5 <https://pytorch.org/hub/ultralytics_yolov5>`_.
     """
 
+    local_name = "ultralytics_yolov5_master"
+    remote_name = "ultralytics/yolov5"
+    model_local_path = f"/home/{getpass.getuser()}/.cache/torch/hub/{local_name}"
+    model_source = "local"
+    model_name = "yolov5s"
+
     def __init__(self) -> None:
         """Initialize local path"""
         self.net = None
-        self.model_local_path = f"/home/{getpass.getuser()}/.cache/torch/hub/ultralytics_yolov5_master"
 
     def load_model(self) -> None:
         """Load model from PyTorchHub"""
         print("Loading YOLOv5 Torch model ...")
         if os.path.exists(self.model_local_path):
```

Comment:
since model_local_path is a class attribute, I think you should reference it using the class name instead of self

Reply:
It wouldn't matter, as assigning a variable as a class attribute is just for code integrity, i.e., this variable should be persisted throughout all created instances from the same class. Moreover, in the class implementation it makes more sense to use `self` rather than the class name to ensure we are calling the class that we are currently developing. If it was on a different module other than the class's, your point would be valid.